src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -84,6 +83,17 @@ public interface IDataBrowserPresenter : IPresenter<IDataBrowserView>, IPresente
       ///    Returns all the DataColumns for the curves that are visible in the chart
       /// </summary>
       IReadOnlyList<DataColumn> GetAllUsedDataColumns();
+
+      /// <summary>
+      /// sets the group row format of the gridView to the specified string.
+      /// </summary>
+      void SetGroupRowFormat(string format);
```

Comment:
I fill this format should be an enum or sthg. You just cannot pass anything there can you?